tests/pytests/functional/modules/cmd/test_script.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+
+import salt.modules.cmdmod as cmdmod
+import salt.modules.cp as cp
+import salt.utils.path
+import salt.utils.platform
+
+if salt.utils.platform.is_windows():
+    import salt.modules.win_file as file
+else:
+    import salt.modules.file as file
+
+pytestmark = [
+    pytest.mark.core_test,
+    pytest.mark.windows_whitelisted,
+]
+
+
+@pytest.fixture(scope="module")
+def configure_loader_modules(minion_opts):
+    return {
+        cmdmod: {
+            "__salt__": {"cp.cache_file": cp.cache_file, "file.remove": file.remove},
+            "__opts__": minion_opts,
+        },
+        cp: {
+            "__opts__": minion_opts,
+        },
+    }
```

Comment:
```suggestion
@pytest.fixture(scope="module")
def configure_loader_modules(minion_opts):
    return {
        cmdmod: {
            "__salt__": {"cp.cache_file": cp.cache_file, "file.remove": file.remove},
            "__opts__": minion_opts,
        },
        cp: {
            "__opts__": minion_opts,
        },
    }
```
These are functional tests, not unit tests, why do you need this?

Reply:
I don't know why I do the things I do... It is now fixed.